pkg/operation/botanist/logging.go

```diff
@@ -20,156 +20,34 @@ import (
 	"path/filepath"
 
 	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
-	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
 	"github.com/gardener/gardener/pkg/client/kubernetes"
 	"github.com/gardener/gardener/pkg/features"
 	gardenletfeatures "github.com/gardener/gardener/pkg/gardenlet/features"
 	"github.com/gardener/gardener/pkg/operation/common"
-	"github.com/gardener/gardener/pkg/utils"
-	kutil "github.com/gardener/gardener/pkg/utils/kubernetes"
-	"github.com/gardener/gardener/pkg/utils/secrets"
-
-	corev1 "k8s.io/api/core/v1"
 )
 
-// DeploySeedLogging will install the Helm release "seed-bootstrap/charts/elastic-kibana-curator" in the Seed clusters.
+// DeploySeedLogging will install the Helm release "seed-bootstrap/charts/loki" in the Seed clusters.
 func (b *Botanist) DeploySeedLogging(ctx context.Context) error {
-	if b.Shoot.GetPurpose() == gardencorev1beta1.ShootPurposeTesting || !gardenletfeatures.FeatureGate.Enabled(features.Logging) {
-		return common.DeleteLoggingStack(ctx, b.K8sSeedClient.Client(), b.Shoot.SeedNamespace)
-	}
-
-	var (
-		kibanaCredentials                      = b.Secrets["kibana-logging-sg-credentials"]
-		kibanaUserIngressCredentialsSecretName = "logging-ingress-credentials-users"
-		sgKibanaUsername                       = kibanaCredentials.Data[secrets.DataKeyUserName]
-		sgKibanaPassword                       = kibanaCredentials.Data[secrets.DataKeyPassword]
-		sgKibanaPasswordHash                   = kibanaCredentials.Data[secrets.DataKeyPasswordBcryptHash]
-		basicAuth                              = utils.EncodeBase64([]byte(fmt.Sprintf("%s:%s", sgKibanaUsername, sgKibanaPassword)))
-
-		sgCuratorPassword     = b.Secrets["curator-sg-credentials"].Data[secrets.DataKeyPassword]
-		sgCuratorPasswordHash = b.Secrets["curator-sg-credentials"].Data[secrets.DataKeyPasswordBcryptHash]
-
-		sgUserPasswordHash  = b.Secrets[kibanaUserIngressCredentialsSecretName].Data[secrets.DataKeyPasswordBcryptHash]
-		sgAdminPasswordHash = b.Secrets[common.KibanaAdminIngressCredentialsSecretName].Data[secrets.DataKeyPasswordBcryptHash]
-
-		userIngressBasicAuth  = utils.CreateSHA1Secret(b.Secrets[kibanaUserIngressCredentialsSecretName].Data[secrets.DataKeyUserName], b.Secrets[kibanaUserIngressCredentialsSecretName].Data[secrets.DataKeyPassword])
-		adminIngressBasicAuth = utils.CreateSHA1Secret(b.Secrets[common.KibanaAdminIngressCredentialsSecretName].Data[secrets.DataKeyUserName], b.Secrets[common.KibanaAdminIngressCredentialsSecretName].Data[secrets.DataKeyPassword])
-		ingressBasicAuth      string
-
-		sgFluentdPasswordHash string
-	)
-
-	userIngressBasicAuthDecoded, err := utils.DecodeBase64(userIngressBasicAuth)
+	err := common.DeleteOldLoggingStack(ctx, b.K8sSeedClient.Client(), b.Shoot.SeedNamespace)
 	if err != nil {
-		return err
+		return nil
```

Comment:
Why not return the error? If there is a valid reason, would you mind adding a short comment to explain? Otherwise somebody might feel compelled to fix this in the future.

Reply:
just a bug, fixed now :)